tools/sync/command.py

```diff
@@ -199,3 +270,6 @@ def getoutput(self):
 
     def getstate(self):
         return self.state
+
+    def getpid(self):
```

Comment:
Maybe `get_pid` or `getPid`?

Reply:
I'd have to change all the other get* methods, maybe some other time.